src/rsa/verification.rs

```diff
@@ -98,26 +177,16 @@ pub fn verify_rsa(params: &RSAParameters,
         return Err(error::Unspecified);
     }
 
-    let n = try!(PositiveInteger::from_be_bytes(n));
-    let e = try!(PositiveInteger::from_be_bytes(e));
+    let n = try!(bn::PositiveInteger::from_be_bytes(n));
```

Comment:
Those lines definitely _could've_ been in the other commit.

Before my first commit, `PositiveInteger` was used unqualified everywhere. I didn't want to change that in _that_ commit because I was trying to make it clear I was just moving existing code over into a new module.

It may just be better to leave the `bn` _types_ as unqualified imports. What do you think?